### ISSUE TITLE
Regression: `bracketName(";")` parameter broken.

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplates#219]
Since version 3.15, the following template doesn't insert the name correctly:

```
%from(bracketName(";"),bracketMail(),toclipboard)%
%header.set(to,clipboard)%
```

Apparently when copying to clipboard the name portion is malformed, creating the following result:

`<Thunder Daily> <thunderbirddaily@gmail.com>`

this will insert a faulty address into the `to` header and as such fail.

Expected behavior:

`"Thunder Daily" <thunderbirddaily@gmail.com>`

**Comment by RealRaven2000** [comment]
As a temporary workaround, one can use:

`%from(bracketName(';'),bracketMail(),toclipboard)%`

Thunderbird accepts single quotes around the name portion and will deliver the email as expected.

**Comment by RealRaven2000** [comment]
Found the problem. New version below:

[smartTemplate-fx-3.15.2pre1.zip](https://github.com/RealRaven2000/SmartTemplates/files/10104247/smartTemplate-fx-3.15.2pre1.zip)

To test, download the zip file and drag into Thunderbird Add-ons Manager (don't extract contents).

**Comment by SILENT-da** [comment]
I tested it, and it works! - issue is solved.
Thank you very much.

**Comment by RealRaven2000** [comment]
> I tested it, and it works! - issue is solved. Thank you very much.

thanks for getting back to me here - you can always raise new issues here on Github - I will be automatically notified of all activity here and this is also the main place for uploading new test versions before features get released officially. I close related comments once they are released in a public version.

**Comment by RealRaven2000** [comment]
Fixed in version 3.16 published on 6th or March 2023
